Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -2098,6 +2117,39 @@ public static bool TryGetObjectAlignedBoundsSize(Transform obj, out Vector3 alig
             return (canGetSize) ? true : false;
         }
 
+        /// <summary>
+        /// Safely adds a child game object to scroll collection.
+        /// </summary>
+        public bool AddItem(GameObject item)
```

Comment:
i'm wondering if it would make sense to have a AddItem with a list of gameobjects as well - just so if several elemetns were added (or removed) we wouldn't call Reset all the time.

Reply:
I think it would Bernie. This method is only covering a very unlikely situation of objects being added on runtime with interactions happening at the same time.... so yeah... adding a collection of objects make sense

Reply:
> Did you think about splitting releaseThresholdLeftRight and TopBottom like you did with back and front? would be more consistent but not sure if it's needed. 

Yeah I was unsure, as at least in the shell top and bottom always match, same as left and right. But front is how the click happens so this will always be different from back. But we could separate them.

> Also what we could consider for future work (can be after graduation) is to expose these thresholds as planes so users can easier visualize them in the editor when setting them up - have a look at the debug planes in PressableButton. if you think it would make sense we could ask a task for that for later

Second that. This one was on my list as well